modules/arm_plugin/src/transformations/quantize_fusion.cpp

```diff
@@ -387,6 +390,9 @@ ArmPlugin::pass::DequantizeInputFusion::DequantizeInputFusion() {
             }
 
             auto node = output->output(0).get_target_inputs().begin()->get_node()->shared_from_this();
+            if (ngraph::is_type<opset::AvgPool>(node) && node->get_input_shape(0).size() == 5) {
```

Comment:
It would be better to move this check inside the first if branch.
Network could contain unquantized AvgPool layer that at the moment will be processed by the second branch and prepended with Dequantize layer instead of scale and/or shift layers